source/dlangbot/github.d

```diff
@@ -51,6 +51,11 @@ string markdownEscape(string desc)
     return app.data;
 }
 
+bool usesGitHubIssues(in ref PullRequest pr)
```

Comment:
normally I would do `usesBugzillaIssues` but I don't know what the full set of repos that use it are.

Reply:
Were you looking for this? 

https://github.com/dlang/dlang-bot/blob/fddfa3545bd699176c812012159932c1a627de28/source/dlangbot/bugzilla.d#L8-L10